### PR TITLE
Magecloud 2573 admin variables

### PR DESCRIPTION
**Pull request by meker12**
## This PR is a:

- [ ] New topic
- [x ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary
- You can no longer use environment variables (`ADMIN_EMAIL`, `ADMIN_USERNAME`, `ADMIN_PASSWORD`) to override admin credentials during upgrades. If you cannot access the Admin panel, use the *Forgot password* feature or the Magento CLI `admin:user:create` command to create a new admin user. See [Access your Magento Admin panel](https://devdocs.magento.com/guides/v2.3/cloud/onboarding/onboarding-tasks.html).
- ADMIN_EMAIL is no longer required when upgrading or applying patches.

## Additional information

*Affected URLs*
- https://devdocs.magento.com/guides/v2.3/cloud/before/before-setup-env-2_clone.html#change-the-magento-admin-variables
- https://devdocs.magento.com/guides/v2.2/cloud/env/environment-vars_magento.html
- https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-install.html
- https://devdocs.magento.com/guides/v2.2/cloud/release-notes/cloud-tools.html

whatsnew
*ADMIN variable behavior changes*
-  You can no longer use environment variables (`ADMIN_EMAIL`, `ADMIN_USERNAME`, `ADMIN_PASSWORD`) to override admin credentials during upgrades. If you cannot access the Admin panel, use the *Forgot password* feature or the Magento CLI `admin:user:create` command to create a new admin user. See [Access your Magento Admin panel](https://devdocs.magento.com/guides/v2.3/cloud/onboarding/onboarding-tasks.html).
-  ADMIN_EMAIL is no longer required when upgrading or applying patches.

### CODE REVIEW

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 1]
```diff
@@ -8,16 +8,28 @@ functional_areas:
 
 Users that have administrative access to the {{site.data.var.ece}} project can use the
 following project environment variables to override the configuration settings
-for the administrative user account and Admin panel. See [Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
+for the administrative user account to access the Admin UI. See [Change the Magento ADMIN variables]({{ page.baseurl }}/cloud/before/before-setup-env-2_clone.html#change-the-magento-admin-variables).
 
 
-| Variable        | Description                                                                                                                                                                                                                                                                                            |
+**Admin credentials**
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]: H2 or 3?

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 2]
```diff
@@ -8,16 +8,28 @@ functional_areas:
...
+| `ADMIN_USERNAME`|A username for the administrative user with the ability to create other users, including administrative users. During initial project provisioning, the username defaults to the {{ site.data.var.ece }} License Owner email address. You can use the default value, or change it to a secure username.|
+|`ADMIN_EMAIL`|Email address for the administrative user. This address is used to send password reset notifications.|
+|`ADMIN_PASSWORD`|Password for the administrative user. When the project is created a random password is generated and an email is sent to the {{ site.data.var.ece }} License Owner. During project creation, the License Owner should have already changed the password. You might need to contact the License Owner for the updated password.|
+| `ADMIN_LOCALE`    | Default—`en_US` Specifies the default locale used by the Magento Admin.
 {:style="table-layout:auto;"}
+
+
+**Admin URL**
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]: H2 or 3?

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 3]
```diff
@@ -8,16 +8,28 @@ functional_areas:
...
+
+Use the following environment variable to secure access to your Magento Admin UI. If specified, this value overrides the default URL during installation.
+
+**`ADMIN_URL`**—The relative URL to access the Magento Admin UI. For example: `/admin`. For security reasons, we recommend that you choose a value other than `admin`,`backend`, or another term that is easy to guess.
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]:
```suggestion
**`ADMIN_URL`**—The relative URL to access the Magento Admin UI. For example: `/admin`. For security reasons, we recommend that you choose a value other than `admin`, `backend`, or another term that is easy to guess.
```

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 4]
```diff
@@ -8,16 +8,28 @@ functional_areas:
...
-for the administrative user account and Admin panel. See [Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
+for the administrative user account to access the Admin UI. See [Change the Magento ADMIN variables]({{ page.baseurl }}/cloud/before/before-setup-env-2_clone.html#change-the-magento-admin-variables).
 
 
-| Variable        | Description                                                                                                                                                                                                                                                                                            |
+**Admin credentials**
+
+Use the ADMIN variables in the following table to override credentials for the Magento Admin user during installation. 
+
+If you want to change the values after installation, connect to your environment using ssh and use the the Magento CLI [`admin:user` command`]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-admin.html) to create or edit the Magento Admin user credentials.
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]:
```suggestion
If you want to change the values after installation, connect to your environment using ssh and use the the Magento CLI [`admin:user` command]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-admin.html) to create or edit the Magento Admin user credentials.
```

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 5]
```diff
@@ -8,16 +8,28 @@ functional_areas:
...
 {:style="table-layout:auto;"}
+
+
+**Admin URL**
+
+Use the following environment variable to secure access to your Magento Admin UI. If specified, this value overrides the default URL during installation.
+
+**`ADMIN_URL`**—The relative URL to access the Magento Admin UI. For example: `/admin`. For security reasons, we recommend that you choose a value other than `admin`,`backend`, or another term that is easy to guess.
+
+If you need to change the Admin URL after installation, connect to your environment using ssh and use the Magento CLI [`magento setup:config`]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-deployment.html) command to change the URL.
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]: I just noticed this command is highlighted differently than the previous one for admin:user.

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 6]
```diff
@@ -8,16 +8,28 @@ functional_areas:
...
-| ADMIN_USERNAME  | Username for the administrative user. The administrative user can create other users, including other administrative users. The default hardcoded username is the Project Owner email address. You can use this value, or change it to another secure username.                                        |
+| `ADMIN_USERNAME`|A username for the administrative user with the ability to create other users, including administrative users. During initial project provisioning, the username defaults to the {{ site.data.var.ece }} License Owner email address. You can use the default value, or change it to a secure username.|
+|`ADMIN_EMAIL`|Email address for the administrative user. This address is used to send password reset notifications.|
+|`ADMIN_PASSWORD`|Password for the administrative user. When the project is created a random password is generated and an email is sent to the {{ site.data.var.ece }} License Owner. During project creation, the License Owner should have already changed the password. You might need to contact the License Owner for the updated password.|
+| `ADMIN_LOCALE`    | Default—`en_US` Specifies the default locale used by the Magento Admin.
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]: If there are default values, it may help the table to have a column for Defaults... ?

Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]: Remove "Specifies" to match the others.

**Comment by meker12** [comment]
Fixed the formatting issues and added default column to table.

**Comment by hguthrie** [comment]
BTW, it looked like a Fastly file update got in there...

**Pull request merged**